x-pack/plugins/security/server/routes/authorization/privileges/get.ts

```diff
@@ -38,12 +38,10 @@ export function defineGetPrivilegesRoutes({ router, authz }: RouteDefinitionPara
             space: Object.keys(privileges.space),
             features: Object.entries(privileges.features).reduce(
               (acc, [featureId, featurePrivileges]) => {
-                return {
-                  ...acc,
-                  [featureId]: Object.keys(featurePrivileges),
-                };
+                acc[featureId] = Object.keys(featurePrivileges);
+                return acc;
               },
-              {}
+              {} as Record<string, unknown>
```

Comment:
__nit:__ if I read it correctly
```suggestion
              {} as Record<string, string[]>
```